redesign/simple_server.js

```diff
@@ -0,0 +1,14 @@
+var liveServer = require("live-server");
```

Comment:
YAGNI. 
The command described above means you can avoid having an extra file. 👍